ruby/anagram.rb

```diff
@@ -0,0 +1,16 @@
+class Anagram
+  attr_reader :root_word
```

Comment:
Is this `attr_reader` necessary? I see that you are creating an instance variable of `@root_word` but calling the `root_word` method in Anagram#match.